src/components/LoginDialogueBox.tsx

```diff
@@ -22,8 +64,40 @@ const LoginDialogueBox = (props: DialogueProps) => {
     setPassword(e.target.value);
   };
 
+  const handleSubmit = async (e: SyntheticEvent) => {
+    e.preventDefault();
+    if (props.isVolunteer) {
+      let volunteerData = await loginVol({ variables: { username, password } });
+      if (volunteerData.data.ok) {
+        client.writeData({
+          data: {
+            volunteerID: volunteerData.data.loginVolunteer.volunteer.id,
+            token: volunteerData.data.loginVolunteer.token,
+          },
+        });
+        history.push('/volunteer-dashboard');
+      }
+    } else {
+      console.log('nonprofit');
```

Comment:
probably want to remove.  Where's your eslint rule!?

Reply:
I don't think I have eslint set up